tests/glibc-tests/tst-cond12.c

```diff
@@ -16,6 +16,7 @@
    License along with the GNU C Library; if not, see
    <http://www.gnu.org/licenses/>.  */
 
+#define _GNU_SOURCE
```

Comment:
I have tradtionally done this in the Makefile via:
-D_GNU_SOURCE
But.... I'm not coming up with a reason why one way is better than the other, so we'll go with this.